sql3/planner/expression.go

```diff
@@ -57,6 +64,13 @@ func coerceValue(sourceType parser.ExprDataType, targetType parser.ExprDataType,
 				return nil, sql3.NewErrInternalf("unexpected value type '%T'", value)
 			}
 			return pql.NewDecimal(int64(val)*int64(math.Pow(10, float64(t.Scale))), t.Scale), nil
+		case *parser.DataTypeTimestamp:
```

Comment:
I would expect us to allow int64, but also to accept strings that contained timestamps, but this seems to only accept int64, and NOT accept strings of timestamps?

Reply:
We do accept strings as input to timestamp, this existing functionality starts at line 82, case section handling string source type and timestamp target type.

Reply:
I always forget that coerceValue is switching on source type first, target type second.